### ISSUE TITLE
A11y_Ability Summit Phase 1Digital Venue_Homepage_Video Player_Screen reader: After entering into full screen, voiceover focus is moving in background of home page.

### ISSUE THREAD

**Issue by PadminiGariki** [google/shaka-player#8865]
### GitHub Tags:
#A11y; #A11yTCS; #A11yWCAG2.0; #IOS; #WCAG2.1; #Full Test Pass; #WCAG1.3.1; #Info and relationships; #Ability Summit 2025; #Ability summit Phase 1; #Microsoft Ability Summit; A11ySev3;

"[Check out Accessibility Insights](https://nam06.safelinks.protection.outlook.com/?url=https://accessibilityinsights.io/&data=02%7c01%7cv-stfe%40microsoft.com%7cb67b2c4b646d4f9561a208d6f4b5c39b%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c636965458850501301&sdata=mxhokIKNMb22llsjXHLgU3XZibj1Qfx37rpY4PU2sfE%3D&reserved=0)! - Identify accessibility bugs before check-in and make bug fixing faster and easier." 
### Environment Details: 
Application Name: Ability Summit Phase 1 Digital Venue 
#URL: https://aka.ms/AbilitySummit-Phase1-UAT 
iOS Version: 18.3 
Screen reader: Voiceover 
### Repro Steps: 

1. Hit the URL: https://aka.ms/AbilitySummit-Phase1-UAT. 
2. Ability Summit Homepage will open. 
3. Turn on voiceover.  
4. Swipe to video player. 
5. Now swipe to Full screen button. 
6. Observe that voiceover focus is moving in back home page 

### Actual: 
After entering into full screen, voiceover focus is moving in back home page 
### Expected: 
After entering into full screen, voiceover focus should be in Full screen page.  
### User Impact: 
Screen reader users will not be able to access the controls in full screen mode and may face the issues.
### WCAG Reference: 
[WCAG 1.3.1- Info and Relationships](https://www.w3.org/WAI/WCAG21/Understanding/info-and-relationships.html)

### Attachment:

[8865.zip](https://github.com/user-attachments/files/21160457/8865.zip)

**Comment by avelad** [comment]
This issue does not follow the structure of issues created for the repository and the attachment does not show anything from Shaka.

**Comment by PadminiGariki** [comment]
Hi @avelad 
why the bugs got closed could you please provide us the comments
